packages/indexer-agent/src/commands/start.ts

```diff
@@ -774,7 +774,7 @@ export default {
       logger,
       network,
       models: queryFeeModels,
-      collectEndpoint: new URL(argv.collectReceiptsEndpoint),
+      collectEndpoint: argv.collectReceiptsEndpoint,
```

Comment:
Would it make sense to use a more general collectEndpoint? or if we could use `--network-subgraph-endpoint` to build `collectEndpoint`, `partialVoucherEndpoint`, and `voucherEndpoint`?

Reply:
I think it's ok to keep the separate specific endpoints for now.